### PR TITLE
Give more details about the default transition on inclusive gateway

### PR DESCRIPTION
**Pull request by alachambre**
[BST-151](https://bonitasoft.atlassian.net/browse/BST-151)

### CODE REVIEW

File: md/gateways.md [hunk 1]
```diff
@@ -22,4 +22,8 @@ several outputs defined, a condition is required to determine which one fires.
   ![diagram of a inclusive gateway](images/images-6_0/papde_pm_diag_gateways_inclusive_gate.png)
 
 An inclusive gateway waits for the input from all active paths, and activates all outgoing transitions that have a condition that
-evaluates to true. If there is only one outgoing transition, no condition is needed and the transition is always activated. If there is more than one outgoing transition, every transition must either be the default or must have a condition.
+evaluates to true.  
+If there is only one outgoing transition, no condition is needed and the transition is always activated.  
+If there is more than one outgoing transition, every transition must either be the default or must have a condition.  
+The default transition is used if all other conditions are evaluated to false. It is not mandatory to have a default transition, but it is a way to ensure that your process won’t end successfully in the middle of its flow because all conditions returned false.  
+If you don't want to define a default transition, you should ensure that at least one of your conditions will be evaludated to true.
```
Comment (baptistemesta) [inline md/gateways.md]:
```suggestion
If you don't want to define a default transition, you should ensure that at least one of your conditions will be evaluated to true.
```

File: md/gateways.md [hunk 2]
```diff
@@ -22,4 +22,8 @@ several outputs defined, a condition is required to determine which one fires.
   ![diagram of a inclusive gateway](images/images-6_0/papde_pm_diag_gateways_inclusive_gate.png)
 
 An inclusive gateway waits for the input from all active paths, and activates all outgoing transitions that have a condition that
-evaluates to true. If there is only one outgoing transition, no condition is needed and the transition is always activated. If there is more than one outgoing transition, every transition must either be the default or must have a condition.
+evaluates to true.  
+If there is only one outgoing transition, no condition is needed and the transition is always activated.  
+If there is more than one outgoing transition, every transition must either be the default or must have a condition.
```
Comment (baptistemesta) [inline md/gateways.md]:
```suggestion
If there is more than one outgoing transition, every transition should either be the default or should have a condition.  
```

File: md/gateways.md (hunk 1 above)
Comment (baptistemesta) [inline md/gateways.md]: After reading again the code:

parallel gateway: 
* all non default transitions are taken (without checking conditions), default transition is not taken (???)

inclusive:
* all transition that have a condition evaluated to "true" or DO NOT HAVE CONDITION (considered as tue) are taken
* if there is none the default transition is taken
* if there is no default an **exception is thrown**

exclusive:
* first transition that have a condition evaluated to "true" or DO NOT HAVE CONDITION (considered as tue) is taken
* if there is none the default transition is taken
* if there is no default an **exception is thrown**

Comment (alachambre) [reply md/gateways.md]: Parallel -> You can't add condition or set as default an outgoing transition of a parallel gateway from the Studio -> all transitions are taken -> make sens

I'll add some details according to your informations, thx

**Pull request merged**